css/shelly/shelly.css

```diff
@@ -0,0 +1,14 @@
+.reamde {
+  text-align: center;
+}
+.center {
+  margin: 0 auto;
+  float: none;
+}
+
+footer {
+  margin-top: 2em;
+  background: white;
+}
+
+{background: url('/img/shelly/DSC03225%20(1).jpg');}
```

Comment:
Curly brackets aren't linked with anything right now. If you change to body { background: ... } that should work.